### PR TITLE
Add epinio-installer helm chart

### PR DESCRIPTION
**Pull request by manno**
The epinio-installer helm chart wraps the epinio helm chart and is able
to install required and optional components.

### CODE REVIEW

File: chart/epinio-installer/templates/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            type: annotation
+
+      - id: registry
+        # needs: namespace-registry
+        needs: cert-manager
+        namespace: epinio-registry
+        type: helm
+        source:
+          name: epinio-registry
+          path: assets/installer/container-registry-0.1.0.tgz
```
Comment (jimmykarily) [inline chart/epinio-installer/templates/manifest.yaml:208]: This now has a release: https://github.com/epinio/epinio-helm-chart/releases/tag/container-registry-0.1.0

Maybe you can add it here: https://github.com/epinio/epinio-helm-chart/pull/19/files#diff-2e442565fbce952d11e55f84855064fdd5e9d981d922df9af960f7b6ef733e22R54

Comment (manno) [reply chart/epinio-installer/templates/manifest.yaml:208]: @jimmykarily is it ok if I remove the file from the installer and switch the chart to the URL?

Comment (manno) [reply chart/epinio-installer/templates/manifest.yaml]: done

File: chart/epinio-installer/values.yaml [hunk 2]
```diff
@@ -50,3 +50,11 @@ domain: "localhost.omg.howdoi.website"
 
 # Use TLS when application images will be pulled by kubernetes. Only applies to the internal registry. (default "false")
 # --force-kube-internal-registry-tls: ""
+
+containerRegistryChart: "https://github.com/epinio/epinio-helm-chart/releases/download/container-registry-0.1.0/container-registry-0.1.0.tgz"
+epinioChart: "https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.1.0/epinio-0.1.0.tgz"
```
Comment (jimmykarily) [inline chart/epinio-installer/values.yaml:55]: You'll need to consume a release that has this in: https://github.com/epinio/epinio-helm-chart/pull/20
(to fix the default namespace not having a serviceaccount)

**Review by jimmykarily — APPROVED** [review]

File: chart/epinio-installer/values.yaml (hunk 2 above)
Comment (jimmykarily) [reply chart/epinio-installer/values.yaml:55]: https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.1.21/epinio-0.1.21.tgz

Comment (jimmykarily) [reply chart/epinio-installer/values.yaml:55]:
```suggestion
epinioChart: "https://github.com/epinio/epinio-helm-chart/releases/download/epinio-0.1.21/epinio-0.1.21.tgz"
```

**Pull request merged**